shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -32,6 +32,18 @@ @interface FlutterViewController () <FlutterBinaryMessenger>
 @property(nonatomic, readwrite, getter=isDisplayingFlutterUI) BOOL displayingFlutterUI;
 @end
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED < 130000
```

Comment:
Since it seems pretty clear from all the discussions that most of the Flutter team is not familiar with the general approach of what is being done in this block, please add a comment explaining that this is declaring things from the iOS 13 SDK so that code in this class can compile even though we're still using the iOS 12 SDK.

Reply:
done